### PR TITLE
=doc release notes for 10.0.6

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
...
+10.0.6 is a security and maintenance release in the stable 10.0.x series of Akka HTTP.
+
+@@@ warning
+
+This release contains a fix for a serious security vulnerability that allows a remote attacker to shut down any Akka
+HTTP application using the routing DSL. See the
+@ref[details](security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md) for more information.
+Please update as soon as possible.
+
+@@@
```
Comment (ktoso) [inline docs/src/main/paradox/release-notes.md]: 👍

File: docs/src/main/paradox/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md [hunk 2]
```diff
@@ -27,11 +27,11 @@ Applications written using only the low-level API from akka-http-core but not th
 
 - akka-http prior to `10.0.6` and  `2.4.11.2`
 
-Notable **not affected**: 
+Notably **not affected**:
 
 - Play Framework (regardless of used server backend)
 - Lagom Framework
-- Users of the low-level akka-http-core APIs
+- Low-level akka-http-core APIs
```
Comment (ktoso) [inline docs/src/main/paradox/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md]: ok

File: docs/src/main/paradox/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Please subscribe to the [akka-security](https://groups.google.com/forum/#!forum/
 
 ## Severity 
 
-The CVSS score of this vulnerability is 7.8 (High), based on vector [AV:N/AC:L/Au:N/C:N/I:N/A:C](https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator?vector=(AV:N/AC:L/Au:N/C:N/I:N/A:C)).
+The CVSS score of this vulnerability is 7.8 (High), based on vector [(AV:N/AC:L/Au:N/C:N/I:N/A:C)](https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator?vector=%28AV:N/AC:L/Au:N/C:N/I:N/A:C%29).
```
Comment (ktoso) [inline docs/src/main/paradox/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md]: ouch, thanks for checking - what a day, all I did was wrong :P Sorry

Comment (jrudolph) [inline docs/src/main/paradox/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.md]: Not sure if that's a paradox bug or a general problem with parentheses in markdown links.

**Pull request merged**